### ISSUE TITLE
Process for actually forking and republishing the DevTools Extension?

### ISSUE THREAD

**Issue by markerikson** [reduxjs/redux-devtools-extension#1]
It appears that the Redux DevTools Extension for Firefox has disappeared from addons.mozilla.org, which is not cool.  We should try to figure out a plan for moving forward with a fork.

The major questions are:

- Package names for publishing things on NPM
- Naming, ownership, and process of publishing extensions for Chrome and FF

For package names, looks like Dan does own `@redux` on NPM, we just need to get him to give us rights to it.  

For extension publishing... I have no idea.

Suggestions?

**Comment by dashed** [comment]
@markerikson would you also be planning on publishing this fork to the chrome web store?

**Comment by markerikson** [comment]
Yes, absolutely - we just need to figure out what the right approach is.

We may just have to label it as the "OFFICIAL Redux DevTools Extension", or something like that, if we can't somehow take ownership of the published extension.

**Comment by markerikson** [comment]
To emphasize something:  @zalmoxisus has done a _fantastic_ job creating and maintaining the Redux DevTools Extension and related packages so far, and I want to give him all the credit for doing so.  Unfortunately, he seems to have almost entirely disappeared from online activity.  Looking at his Github graph, I see two days of activity in August, two days back in March, and that's it for 2018.  That's not healthy for a widely-used extension.

Even if he pops up and says he's about to get involved again, I would still want to go ahead and move the repos into the `reduxjs` Github org.  I would _love_ for him to still be actively involved in maintaining everything, but it's too important a tool to let it sit unmaintained.  (And yes, the fact that I have [an outstanding PR to add "action stack traces"](https://github.com/zalmoxisus/redux-devtools-extension/issues/429#issuecomment-417955282) does play a role in this...)

**Comment by kumar303** [comment]
> For extension publishing... I have no idea.

Hi. The best way to take ownership of https://addons.mozilla.org/en-US/firefox/addon/remotedev/ is ask the original owner to sign into https://addons-dev.allizom.org/en-US/developers/ and add you as an owner:

<img width="764" alt="screenshot 2018-10-31 09 40 23" src="https://user-images.githubusercontent.com/55398/47796484-70f37580-dcf2-11e8-9641-bf27d21d93f8.png">

Sorry about the confusion. There were problems with some recent versions of the extension and addons.mozilla.org reviewers could not get in touch with the owner after many attempts. We have restored it to a previous version.

**Comment by vcarl** [comment]
> reviewers could not get in touch with the owner after many attempts

Seems to be the theme :) I mentioned this in Reactiflux a few weeks ago, but IMO the "remotedev" org is more confusing than valuable. It would be nice to continue using the original package, but I think it would be good to publish one with the Redux logo under a "Redux DevTools" org.

**Comment by markerikson** [comment]
FWIW, NPM has handed over the `reduxjs` org to me, and I've added Tim and Dan.

Dan still has control over the `redux` org.  I keep poking him to add us, and he keeps forgetting :)  I'd prefer to use that org long-term for publishing stuff - ie, `@redux/devtools-extension`.

**Comment by zalmoxisus** [comment]
Hey guys. Sorry for being out. Let me know what I can help with this. We have a group for Chrome extension publishing, where @jhen0409 and I have access. Didn't look if that's possible for Mozilla too. If you need something asap, you can reach me at zalmoxisus[at]gmail.
UPDATE: I see the message above, so looks like can easily add you there. And I'll move all the repos you think it's necessary to redux organization, but probably a monorepo would be better, since there are many of them.

**Comment by zalmoxisus** [comment]
Also we collected $3k in bounties on OC for those who will work on this, I didn't touch any of those.

**Comment by markerikson** [comment]
@zalmoxisus : hey, great to hear from you!  

My first immediate request would be for you to merge the PRs listed at https://github.com/zalmoxisus/redux-devtools-extension/issues/429#issuecomment-417955282 and republish the FF and Chrome extensions, so we can get the "action stack trace" feature out to everyone ASAP.

After that, we can have a more detailed discussion about how to best organize and maintain the extension going forward.

**Comment by zalmoxisus** [comment]
Firefox extension was reviewed and available now. Also I added access to @jhen0409 there as well, didn't know that's possible (it's easier than using groups for Chrome). Agree that using `remotedev` there is confusing, I used that because at the moment of publishing, `redux-devtools` was used by someone else. I see it looks available now (probably because old extensions with non-chrome api were removed), but no option to change, except publishing it 2 times with different handles. Not sure that is allowed, but can try.

As for moving/forking the extension here, don't think that helps. As you noticed, most of needed changes are outside. Moving at least 20 repositories related to devtools only to reduxjs organization doesn't seem reasonable. The best solution, especially for new contributors, would be to transform [`redux-devtools`](https://github.com/reduxjs/redux-devtools) into a monorepo and adding all packages there. We are talking about:
1. [`redux-devtools-extension`](https://github.com/zalmoxisus/redux-devtools-extension)
2. [`redux-devtools-instrument`](https://github.com/zalmoxisus/redux-devtools-instrument)
3. [`remotedev-app`](https://github.com/zalmoxisus/remotedev-app)
4. [`remotedev-serialize`](https://github.com/zalmoxisus/remotedev-serialize)
5. [`remotedev-utils`](https://github.com/zalmoxisus/remotedev-utils)
6. [`redux-devtools-filter-actions`](https://github.com/zalmoxisus/redux-devtools-filter-actions)
7. [`remote-redux-devtools`](https://github.com/zalmoxisus/remote-redux-devtools)
8. [`devui`](https://github.com/zalmoxisus/devui)
9. [`remotedev-slider`](https://github.com/zalmoxisus/remotedev-slider)
4. [`remotedev-server`](https://github.com/zalmoxisus/remotedev-server)
5. [`redux-remotedev`](https://github.com/zalmoxisus/redux-remotedev)
3. [`atom-redux-devtools`](https://github.com/zalmoxisus/atom-redux-devtools)
10. [`redux-devtools-test-generator`](https://github.com/zalmoxisus/redux-devtools-test-generator)
11. [`redux-devtools-chart-monitor`](https://github.com/reduxjs/redux-devtools-chart-monitor)
12. [`redux-slider-monitor`](https://github.com/calesce/redux-slider-monitor)
13. [`react-json-tree`](https://github.com/alexkuz/react-json-tree)
14. [`redux-devtools-inspector`](https://github.com/alexkuz/redux-devtools-inspector)
15. [`redux-devtools-log-monitor`](https://github.com/gaearon/redux-devtools-log-monitor)
16. [`redux-devtools-themes`](https://github.com/gaearon/redux-devtools-themes)
17. [`redux-devtools-dock-monitor`](https://github.com/gaearon/redux-devtools-dock-monitor)

There are others I could miss or derived like integrations for non-redux apps.
Github provides an option to transfer issues from repositories, of course better to address them, but at least pull requests should be resolved and finished before we move the code and archive those repositories. There's a lot of work in progress. If someone has time to help with that, he's more than welcome.

About the pr mentioned above, we cannot merge and move it to production as is. I addressed https://github.com/zalmoxisus/redux-devtools-instrument/pull/21 as there we not so much to fix/change, but for https://github.com/zalmoxisus/remotedev-app/pull/43 we need to move it as a separate monitor/repository and fix mentioned issues there.

Regarding my github activity, starting from late 2015 till early 2017 you'd hardly find a day not dedicated exclusively to redux-devtools, but life is limited and there's also a lot of other stuff to do)
I understand the concerns as I went through that many times, waiting for months for some monitors to merge my PRs to move forward, then ending to fork them.
